### PR TITLE
Replace README

### LINKED ISSUE #12: Update README
Ceate an initial README for the repository, and remove the auto-generated README.

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #12

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
+= Lisk Documentation
```
Comment (TalhaMaliktz) [inline README.adoc:1]: Why an `.adoc` file and not a `.md` file? 🤔

File: README.adoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This repo houses the Lisk Docs located at https://documentation.lisk.com. All documentation related updates and new content will be tracked and maintained in this repo.
+
+The Lisk documentation uses {url_docusaurus}[Docusaurus^] as a documentation site generator.
+
+== Managing Docs
+=== Local Development
+
+```
+$ yarn start
```
Comment (TalhaMaliktz) [inline README.adoc]: Please remove `$` from all code snippets as it gets copied along with the command as well, and then you have to manually remove it.

```suggestion
yarn start
```

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+=== Build
+
+```
+$ yarn build
+```
+
```
Comment (TalhaMaliktz) [inline README.adoc:38]: Please add the [Testing your Build Locally](https://docusaurus.io/docs/deployment#testing-build-locally) section here as well. I used it a lot as it shows instantly how the documentation site will look. This is important cause, I faced situations while configuring the docs that some settings would work correctly on the local environment but not when you `serve` or `deploy` them. So serving the site locally saved time and hassle of deploying and waiting for `git-pages` to update.

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Please consider the following!

File: README.adoc (hunk 1 above)
Comment (Tschakki) [reply README.adoc:1]: i would ask the other way around :D 
Main reason is I wanted to use autogenerate ToC feature of asciidoc.

File: README.adoc [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+By default, this will load your site at http://localhost:3000/.
+
+=== Build
+
+```
+yarn build
+```
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
```
Comment (TalhaMaliktz) [inline README.adoc]:
```suggestion
This command generates static content into the `build` directory and can be served using any static content hosting service.
```

File: README.adoc [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+=== Deployment
+
+The following command will automatically <<build>> the docs, and push them to the `gh-pages` branch.
+Changes to this branch are automatically deployed under https://liskhq.github.io/lisk-documentation/
+
+Using SSH:
```
Comment (TalhaMaliktz) [inline README.adoc]:
```suggestion
*Using SSH:*
```

File: README.adoc [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+USE_SSH=true yarn deploy
+```
+
+Not using SSH:
```
Comment (TalhaMaliktz) [inline README.adoc]:
```suggestion
*Not using SSH:*
```

**Review by TalhaMaliktz — APPROVED** [review]
Minor suggestions, rest LGTM 👍🏼

**Pull request merged**